CardanoSharp.Wallet/CardanoSharp.Wallet.csproj

```diff
@@ -4,7 +4,7 @@
     <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <LangVersion>9.0</LangVersion>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <Version>1.0.4</Version>
+    <Version>1.0.5</Version>
```

Comment:
Are you not worried about this being a breaking change?

Reply:
this controls the nuget version. nothing more

Reply:
The consumers will likely not expect a breaking change as only the last value in the version is being changed.

Reply:
im not really sure what you mean by all this but the Version is what controls the nuget package version that gets published

Reply:
[https://docs.microsoft.com/en-us/nuget/concepts/package-versioning](https://docs.microsoft.com/en-us/nuget/concepts/package-versioning)

> Major: Breaking changes
> Minor: New features, but backwards compatible
> Patch: Backwards compatible bug fixes only

Reply:
would you not view this more of a bug fix?
![image](https://user-images.githubusercontent.com/638507/160031281-e06983cf-e9b9-44c1-bb23-3df7ff6421fc.png)
https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-basics

Reply:
just to be clear, i honestly dont mind making the version w/e you think is more appropriate

Reply:
Well, yes it is a bug fix, but also breaks code that depends on this being incorrect. 

So for example I'll have to update the following:
![image](https://user-images.githubusercontent.com/57759320/160031466-2bbc5c7c-e986-40a8-a110-1d17ccd489dd.png)

I think if we strictly want to follow the versioning guide it should be a major version increase. But I understand you may have other objectives for major versions and this is a small change with probably very few projects actually depending on the old incorrect implementation.

Reply:
sounds like a good enough reason to me. im competely fine with the call to increment major